applications/dashboard/models/class.usermodel.php

```diff
@@ -1070,6 +1070,9 @@ private function insertInternal($fields, $options = []) {
         $userID = $this->SQL->insert($this->Name, $fields);
 
         if ($userID) {
+            //force clear cache for that UserID
+            $this->clearCache($userID, ['user']);
```

Comment:
If you want to go the distance on this PR you could do the following:

1. Extract that 'user' string to a constant on the class. Do it for the other versions of that string so you would have three `CACHE_*` constants. Hint: PhpStorm has a refactor tool for this.
2. Go ahead and add the `@param` doc block for that argument.

This isn't a change request though, just a comment.